src/sentry/integrations/middleware/hybrid_cloud/parser.py

```diff
@@ -53,22 +52,16 @@ def __init__(
         self.error = error
 
 
-class BaseRequestParser(abc.ABC):
+class BaseRequestParser:
     """Base Class for Integration Request Parsers"""
 
     _METRIC_SUCCESS_KEY = "integrations.middleware.request_parser.success"
     _METRIC_FAILURE_KEY = "integrations.middleware.request_parser.failure"
     _METRICS_INFO_KEY = "integrations.middleware.request_parser.info"
 
-    @property
-    def provider(self) -> str:
-        raise NotImplementedError("'provider' property is required by IntegrationClassification")
-
-    @property
-    def webhook_identifier(self) -> WebhookProviderIdentifier:
```

Comment:
Are we losing doc comments here?

Reply:
I didn't really feel like they added any information that they type annotations didn't add 🤷

Reply:
(or the resulting stacktrace if they were missing)